docs/src/learn/reacting-to-input-with-state.md

```diff
@@ -472,50 +367,23 @@ Make it so that clicking on the picture _removes_ the `background--active` CSS c
 
 Visually, you should expect that clicking on the picture removes the purple background and highlights the picture border. Clicking outside the picture highlights the background, but removes the picture border highlight.
 
-```js
-export default function Picture() {
-	return (
-		<div className="background background--active">
-			<img
-				className="picture"
-				alt="Rainbow houses in Kampung Pelangi, Indonesia"
-				src="https://i.imgur.com/5qwVYb1.jpeg"
-			/>
-		</div>
-	);
-}
-```
+=== "picture.py"
 
-```css
-body {
-	margin: 0;
-	padding: 0;
-	height: 250px;
-}
+	```python
+    {% include "../../examples/python/managing_state/picture_component.py" start="# start" %}
+    ```
 
-.background {
-	width: 100vw;
-	height: 100vh;
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	background: #eee;
-}
+=== "styles.css"
```

Comment:
@Archmonger, we'll need to ensure we should how the CSS is included in our examples